charts/amazon-cloudwatch-observability/templates/linux/neuron-monitor-daemonset.yaml

```diff
@@ -36,7 +36,7 @@ spec:
   - name: PATH
     value: /usr/local/bin:/usr/bin:/bin:/opt/aws/neuron/bin
   - name: GOMEMLIMIT
-    value: 160MiB
+    value: 320MiB
```

Comment:
The default limit itself is lower than this - https://github.com/aws-observability/helm-charts/blob/2711d527e709be50b8a7b7b402f553b30f766fcb/charts/amazon-cloudwatch-observability/values.yaml#L1432

Should we updating that as well?